test/yagna/interactive/conftest.py

```diff
@@ -52,12 +52,3 @@ def test_failure_callback() -> Callable[[TestFailure], None]:
     return lambda error: logger.error(
         f"The runner was stopped due to test failure: {error}"
     )
-
-
-@pytest.hookimpl
-def pytest_report_teststatus(report, config):
-    """Suppress printing test status for interactive testing session."""
-
-    # Return section, one-letter and verbose status, e.g. ("failed", "F", "FAILED").
-    # Returning empty strings will suppress printing status.
-    return "", "", ""  # e.g. "failed", "F", "FAILED"
```

Comment:
This hook was provided just to suppress printing 
```
PASSED

======================================= 1 passed in 77.78s (0:01:17) ========================================
```
But it needs to be more sophisticated in order not to suppress reporting errors in `goth` startup (vs test errors such as step timeout or assertion failure). Let's remove it for now.